From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: DigitalRegent <misterwener@gmail.com>
Date: Sat, 11 Apr 2020 13:10:58 +0200
Subject: [PATCH] Brand support


diff --git a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
<<<<<<< HEAD:Spigot-Server-Patches/0559-Brand-support.patch
index 3846e926e2442bc15dff41c70c5f2176f1c678be..7c8768765346b9349d0110ecf67be0bc8b00254a 100644
--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
@@ -2397,7 +2397,7 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
=======
index fb871480a55d73681f2a45f6f14b984807d51ba3..3f63f83b21d84a4fa1cfda217afb22c4cf7fd2ad 100644
--- a/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
+++ b/src/main/java/org/bukkit/craftbukkit/entity/CraftPlayer.java
@@ -2192,7 +2192,7 @@ public class CraftPlayer extends CraftHumanEntity implements Player {
>>>>>>> master:Spigot-Server-Patches/0556-Brand-support.patch
     // Paper start - brand support
     @Override
     public String getClientBrandName() {
-        return getHandle().playerConnection.getClientBrandName();
+        return getHandle().playerConnection != null ? getHandle().playerConnection.getClientBrandName() : null;
     }
     // Paper end
 
